app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :posts, class_name: 'Post', foreign_key: 'user_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'user_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'user_id'
 end
```

Comment:
- [ ] [REQUIRED] Great implementation so far, however, the User model should also have a method that returns 3 most recent posts.